index.bs

```diff
@@ -2805,6 +2805,11 @@ Attributes</h4>
 				MUST be thrown for any attempt to change the
 				value.</span>
 
+		: {{AudioWorkletNode}}
+		::
+			See <a href="#configuring-channels-with-audioworkletnodeoptions">
```

Comment:
lgtm on the change.  But isn't there a bikeshed way to reference the section?  If so, maybe use that.  Then we should get warnings/errors if we change the section name but forget to update the links.

Reply:
Done. I followed this: https://tabatkins.github.io/bikeshed/#section-links

Reply:
Also confirmed the link is displayed correctly via the preview.